easybuild/easyconfigs/f/FTI/FTI-20141204-intel-2015.02.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Jordi Blasco <jordi.blasco@nesi.org.nz>
+# New Zealand eScience Infrastructure (NeSI)
+
+easyblock = 'CMakeMake'
+
+name = 'FTI'
+version = '20141204'
+
+homepage = 'http://leobago.github.io/fti/'
+description = """FTI stands for Fault Tolerance Interface and is a library that aims to give computational 
+scientists the means to perform fast and efficient multilevel checkpointing in large scale supercomputers. 
+FTI leverages local storage plus data replication and erasure codes to provide several levels of reliability 
+and performance. FTI is application-level checkpointing and allows users to select which datasets needs to be
+protected, in order to improve efficiency and avoid wasting space, time and energy."""
+
+toolchain = {'name': 'intel', 'version': '2015.02'}
```

Comment:
any specific reason to go with this, rather than the 'common' toolchain `intel/2015a`?

it's OK if you're going with this, just wondering if you have a particular reason